cmd/crc/cmd/podman_env.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/machine"
+	"github.com/code-ready/crc/pkg/crc/output"
+	"github.com/code-ready/crc/pkg/os/shell"
+	"github.com/spf13/cobra"
+)
+
+var podmanEnvCmd = &cobra.Command{
+	Use:   "podman-env",
+	Short: "Setup podman environment",
+	Long:  `Setup environment for 'podman' binary to access podman on CRC VM`,
+	Run: func(cmd *cobra.Command, args []string) {
+		userShell, err := shell.GetShell(forceShell)
+		if err != nil {
+			errors.ExitWithMessage(1, "Error running the podman-env command: %s", err.Error())
+		}
+
+		ipConfig := machine.IpConfig{
+			Name:  constants.DefaultName,
+			Debug: isDebugLog(),
+		}
+
+		exitIfMachineMissing(ipConfig.Name)
```

Comment:
Why are we using this name ?

and why not perform this ceck before the IpConfig as `exitIfMachineMissing(constants.DefaultName)` ?

For now we purposely perform all these tests with the DefaultName... IF we would ever move away from this, it would be a simple accross the board search.

Reply:
this code is based on https://github.com/code-ready/crc/blob/master/cmd/crc/cmd/oc_env.go#L26 and that does exactly this as well.

Reply:
Should be refactored as a new issue in that case as bailout conditions should be considered first

Reply:
ok, will revert back then.